### PR TITLE
Fix: Proposal details voting terminal

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- Fix following issue with proposal details: disables vote now if executed/defeated, disables voters tab if no voters yet

Task: [APP-191](https://aragonassociation.atlassian.net/browse/APP-191)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/pages/proposal.tsx [hunk 1]
```diff
@@ -387,3 +395,6 @@ const ContentContainer = styled.div.attrs(
     } mt-3 tablet:flex tablet:space-x-3 space-y-3 tablet:space-y-0`,
   })
 )<ContentContainerProps>``;
+
+const PROPOSAL =
+  "<p><strong>We LOVE pets!</strong><br /></p><p>As we all know, pets are amazing, and the feelings we experience because of their amazing existence are overwhelmingly powerful. But, of course, with great power comes great responsibility, and let's face it: many of us are not paragons of responsibility. We often need to set reminders for doggy bath time or kitty vet visit. </p><br /><p>In order to make sure that our pets are well taken cared of and that we enjoy the awesomeness that is adopting and raising a pet, I propose we create a sub DAO. </p><p>The new Pets United sub DAO would be an organization of pet owners (obviously) looking out for each other, and making sure that everyone is aware of new discounts at the groomer and that we present a united front when eventually we demand<strong> full citizenship </strong>for our pets!</p><br /><p>Looking forward to your comments!</p>";
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/proposal.tsx:400]: Oh, will we be allowing HTML in our metadata? I thought it was going to be MD.

**Review by WissenIstNacht — APPROVED** [review]
Looking good, thanks.

File: packages/web-app/src/pages/proposal.tsx (hunk 1 above)
Comment (RakeshUP) [reply packages/web-app/src/pages/proposal.tsx:400]: That's what the Tiptap rich text editor outputs in HTML mode, but it also has a JSON mode which we should explore. [https://tiptap.dev/guide/output](https://tiptap.dev/guide/output)

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/pages/proposal.tsx (hunk 1 above)
Comment (carlosgj94) [reply packages/web-app/src/pages/proposal.tsx:400]: We are using their official Standard and Link kits which sanitises the inputs. We should be okey

Comment (WissenIstNacht) [reply packages/web-app/src/pages/proposal.tsx:400]: All good, thanks for the explanation 👍🏼

**Pull request merged**